docs/modules/concepts/pages/mail/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Email Domain Model
+:navtitle: Emails
+
+The https://en.wikipedia.org/wiki/History_of_email[history of email] has resulted
+in a convoluted patchwork of standards and practices, which can be very difficult
+to understand and decipher. Fortuneately, James has your back.
+
+We have built up our own view of emails that allows us to process messages in a
+consistent and comprehensible way.
+
+To help you understand how James approaches emails, we have divided this section
+up into the following parts:
+
+ * xref:mail/mime.adoc[Message Contents] (MIME)
+ * xref:mail/smtp.adoc[Message Envelopes] (SMTP)
```

Comment:
Message Transport would seem more correct to me here

Reply:
Good point!

Reply:
Also we should add a section about JMAP (https://jmap.io/) (but I agree to contribute it later on)

Reply:
I can add a placeholder now.

Reply:
Made suggested changes, including additional of stub for JMAP.